java/fury-core/src/main/java/org/apache/fury/ThreadLocalFury.java

```diff
@@ -265,6 +267,16 @@ public ClassLoader getClassLoader() {
     return bindingThreadLocal.get().getClassLoader();
   }
 
+  @Override
+  public ClassChecker getClassChecker() {
```

Comment:
Please implement this in AbstractThreadSafeFury

Reply:
ok

Reply:
> Please implement this in AbstractThreadSafeFury

AbstractThreadSafeFury does not have bindingThreadLocal, so it seems unable to implement this interface.

Reply:
You can invoke processCallback instead

Reply:
> You can invoke processCallback instead

processCallback ReturnType is void

Reply:
I think you can create ClassChecker first and set it to Fury. Then you don't need to add any API to Fury. We won't add such API to get Classchecker or ClassResolver/SerializerFactory

Reply:
> We can only allow set ClassChecker

ok